tests/test_v1.py

```diff
@@ -20,17 +20,9 @@
 import json
 
 from io import BytesIO
-import src
 from src.api import v1
 
 
-def test_home(mongo_app):
```

Comment:
Shouldn't we test here that a user gets redirected?

Reply:
Good idea, unit test added